toolchains/gcc/defs.bzl

```diff
@@ -102,16 +111,17 @@ def download_gcc_distribution(
         version = release.version,
         arch = arch,
         os = os,
+        target_arch = target_arch,
+        target_os = target_os,
     )
 
 def _gcc_distribution_impl(ctx: AnalysisContext) -> list[Provider]:
-    dest = ctx.actions.declare_output("gcc-arm-none-eabi")
+    dest = ctx.actions.declare_output("gcc-{}-none-{}".format(ctx.attrs.target_arch, ctx.attrs.target_os))
```

Comment:
I _think_ the `none` here should actually be OS?

Reply:
its arch-vendor-os-abi as I understand. Youre right though im not the biggest fan of how i went about it because _technically_ i should have been propogating the triples through, but since we never change vendor, os-abi are essentially a software platform and arch is the hardware platform im not fighting to clean it up right away